pages/pipelines/notifications.md

```diff
@@ -299,6 +299,9 @@ steps:
 ```
 {: codeblock-file="pipeline.yml"}
 
+>🚧
+> We cannot substitute `user` with the build creator environment variable value
```

Comment:
Make this more about them than us + add a period:

```suggestion
> You cannot substitute `user` with the build creator environment variable value.
```

Reply:
Is there anything we can say about why this is the case? You mentioned environment variable interpolation in the PR description.